backend/src/plugins/AFK/events/AFKNotificationEvt.ts

```diff
@@ -0,0 +1,28 @@
+import { sendUserMentionMessage, sendWelcomeBackMessage } from "../functions/buildAFKMessages";
+import { afkEvt } from "../types";
+
+export const AFKNotificationEvt = afkEvt({
+    event: 'messageCreate',
+
+    listener: async ({ pluginData, args: { message } }) => {
+        // Mention Check (if someone mentions the AFK user)
+        if (message.mentions.length) {
+            const afk = await pluginData.state.afkUsers.getUserAFKStatus(message.mentions[0].id);
```

Comment:
Might not always be the first mention

Reply:
Yeah, I'll try to make a better system for mention parsing in a bit